content/en/tracing/error_tracking/_index.md

```diff
@@ -1,39 +1,50 @@
 further_reading:
-- link: "/tracing/error_tracking/explorer"
-  tag: "Documentation"
-  text: "Error Tracking Explorer"
-- link: "https://app.datadoghq.com/apm/error-tracking"
-  tag: "UI"
-  text: "Error tracking"
 - link: "https://www.datadoghq.com/blog/service-page/"
   tag: "Blog"
-  text: "Service Telemetry, Error Tracking, SLOs and more"
+  text: "Explore a centralized view into service telemetry, Error Tracking, SLOs, and more"
+- link: "/tracing/error_tracking/explorer"
+  tag: "Documentation"
+  text: "Learn about the Error Tracking Explorer"
+- link: "/tracing/trace_explorer/trace_view/"
+  tag: "Documentation"
+  text: "Learn about the Trace Explorer"
 ---
 
-{{< img src="tracing/error_tracking/explorer_with_backend_issues.png" alt="Error Tracking Page"  >}}
+## Overview
 
+It is critical for your system's health to consistently monitor the errors collected by Datadog. When there are many individual error events, it becomes hard to prioritize errors for troubleshooting. By tracking, triaging, and debugging stack traces, you can minimize the impact of fatal errors on your backend services.
 
-Monitoring the errors collected by Datadog is critical to your system's health, but there can be so many individual error events that it’s hard to identify which ones matter the most and should be fixed first. Error Tracking makes it easier to monitor errors by:
+{{< img src="tracing/error_tracking/explorer_with_backend_issues.png" alt="The Error Tracking Explorer for APM displaying issues from your backend services" style="width:100%;" >}}
 
-- __Grouping similar errors into issues__ to turn this noisy flow of errors into a small list of manageable issues.
-- __Following issues over time__ to show when they first started, if they are ongoing, and how often they are occurring, helping you identify the most important ones.
-- __Getting all the context in one place__ to facilitate troubleshooting.
+## Use span tags to track error spans
 
-## How Datadog error tracking works
+<div class="alert alert-info">Error Tracking is available for all the languages supported by APM and does not require using a different SDK.</div>
 
-The Datadog tracers collect errors through integrations and manual instrumentation of the source code. Error spans within a trace are processed by Error Tracking __when they are located in the uppermost service span__, which is also called the _service entry span_. The span must also contain the `error.stack`, `error.msg`, and `error.type` [span tags][1] in order to be tracked.
+The Datadog tracers collect errors through integrations and the manual instrumentation of your backend services' source code. If an error span within a trace is processed by Error Tracking **when it is located in the uppermost service span**, it is called a _service entry span_. This span must contain the `error.stack`, `error.message`, and `error.fingerprint` [span tags][1] to be tracked.
```

Comment:
No this isn't right. The uppermost service span is called a _service entry span_ regardless of whether there's an error in it. How about:
```suggestion
The Datadog tracers collect errors through integrations and the manual instrumentation of your backend services' source code. Error spans within a trace are processed by Error Tracking **if the error is located in a service entry span** (the uppermost service span). This span must also contain the `error.stack`, `error.message`, and `error.fingerprint` [span tags][1] to be tracked.
```